### PR TITLE
Adds non-copy-code-blocks.

### PR DESCRIPTION
**Pull request by johnnymatthews**
This PR is a port of https://github.com/filecoin-project/lotus-docs/pull/282.

The site now looks for `<pre>` tags that have an immediate sibling that is another `<pre>` tag. If there is one, the reduces the `marginBottom` to `0`. Then on the next element (the second `<pre>` tag, it removes the copy button, sets the `marginTop` to `0`, and reduces the opacity to `0.8`.

This now means that if you've got a codeblock with an output, then you need to put the output in it's own codeblock immediately after the codeblock with the input:

````markdown
1. Install the serve package:

    ```shell
    npm install serve
    ```

    ```shell
    npm installing blah blah blah
    whatever npm outputs these days
    honestly, nobody ever reads npm output anyway.
    ```

1. Run the serve thingy:

    ```shell
    serve
    ```

    You can also run serve with a path:

    ```shell
    serve ./public
    ```
````

This outputs this:

image

### CODE REVIEW

**Comment by mishmosh** [comment]
@johnnymatthews @jennijuju bump, who should review?

**Comment by jennijuju** [comment]
@longfeiWan9 could you please review? 

(fyi @johnnymatthews is OOO until the following week)

**Comment by longfeiWan9** [comment]
> 

Definitely, I will review it today.

**Comment by longfeiWan9** [comment]
It does not seem working after I merged the main into this branch. 

- There are no copy functions for code blocks.
- Code output is still a separate block. 

I will try to fix it.

File: content/en/get-started/store-and-retrieve/troubleshooting/index.md
Comment (longfeiWan9) [inline content/en/get-started/store-and-retrieve/troubleshooting/index.md:37]: This looks not right when I test it. 
It look like 
```
    ```shell
    sysctl -n machdep.cpu.brand_string
    ```
```

File: content/en/get-started/store-and-retrieve/troubleshooting/index.md
Comment (longfeiWan9) [inline content/en/get-started/store-and-retrieve/troubleshooting/index.md:26]: We should not add an extra tab/space here. Because tab/space is already recognized as code-block for MD which will make it looks like:
image

**Review by longfeiWan9 — APPROVED** [review]
I have fixed some tiny issues.

- copy buttons do not appear as they should be 
- double display code-block by adding tab/space

Now, it looks good to me and we can merge it.

**Comment by longfeiWan9** [comment]
@jennijuju @mishmosh I am not authorized to merge this PR. Could you please merge it? Thank you

**Comment by mishmosh** [comment]
@longfeiWan9 you now have merge access to this repo.

**Pull request merged**
